src/pages/Subscriptions/Subscriptions.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import styled from 'styled-components'
+import { useTranslation } from 'react-i18next'
+import { IonContent } from '@ionic/react'
+import { Grid } from '@material-ui/core'
+import SubscriptionFree from './SubscriptionFree'
+import SubscriptionBusiness from './SubscriptionBusiness'
+import SubscriptionPremium from './SubscriptionPremium'
+
+const MainView = styled.div`
+    h1 {
+        font-size: ${({ theme }) => theme.spacing.L};
+        text-align: center;
+    }
+`
+
+export default function Subscriptions() {
+    const { t } = useTranslation('pricing')
+
+    return (
+        <IonContent style={{ '--background': 'white' }}>
```

Comment:
is it still current?

Reply:
no, same as above

Reply:
sorry indeed I'd forgot it, I removed it